JustFakeIt/FakeServer.cs

```diff
@@ -4,13 +4,15 @@
 using System.Linq;
 using Microsoft.Owin.Hosting;
 using Owin;
+using System.Net.Http;
 
 namespace JustFakeIt
 {
     public class FakeServer : IDisposable
     {
         public Uri BaseUri { get; private set; }
         public Expect Expect { get; protected set; }
+        public HttpClient Client { get; private set; }

```

Comment:
If this is just for the tests then I'm not convinced it should be added here, could be moved to a helper somewhere in the test project. If someone is after a HttpClient mock those exist and this isn't one of them, as they'd probably expect it to be in-memory.